DigitalOffice/Tests/UserServiceTests/Commands/GetUserByEmailCommandTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Commands;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.Models;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.UserService.Validators;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.UserServiceUnitTests.Commands
+{
+    public class GetUserByEmailCommandTests
+    {
+        private Mock<IUserRepository> repositoryMock;
+        private Mock<IMapper<DbUser, User>> mapperMock;
+        private Mock<IValidator<string>> validatorMock;
+
+        private IGetUserByEmailCommand command;
+
+        private string userEmail;
+        private User user;
+        private DbUser dbUser;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            userEmail = "example@gmail.com";
+            user = new User { Email = userEmail };
+            dbUser = new DbUser { Email = userEmail };
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            validatorMock = new Mock<IValidator<string>>();
+            repositoryMock = new Mock<IUserRepository>();
+            mapperMock = new Mock<IMapper<DbUser, User>>();
+
+            command = new GetUserByEmailCommand(validatorMock.Object, repositoryMock.Object, mapperMock.Object);
```

Comment:
Add verifying that repository method was not called

Reply:
Fixed